src/App.tsx

```diff
@@ -163,11 +163,15 @@ export default function App() {
     const playerScore = score(hand).value;
     if (playerScore > dealerScore) {
       setMessage(Message.playerWin);
-    } else if (dealerScore > playerScore) {
+    } else if (dealerScore > playerScore && dealerScore <=21) {
       setMessage(Message.dealerWin);
-    } else {
+    } else if (dealerScore <=21 && dealerScore == playerScore) {
       setMessage(Message.push);
     }
+    else
+    {
+        setMessage(Message.playerWin)
```

Comment:
`✏️ suggestion:` There is already a condition above for the `playerWin` resolution; I think it would be better to make a dealer bust part of that condition and keep the `else` falling through to a push, rather than checking that the dealer didn't bust twice and falling through to a redundant resolution.